owsi-core/owsi-core-components/owsi-core-component-wicket-more/src/main/java/fr/openwide/core/wicket/more/rendering/Renderer.java

```diff
@@ -652,5 +654,116 @@ public String render(final T value, Locale locale) {
 			return Localizer.get().getString(resourceKeyBuilder.toString(), null, mapModel, locale, null, null);
 		}
 	}
-
+	
+	/**
+	 * <p>A simplified version of the {@link #range(String, Renderer, Renderer)} function.
+	 * <p>Returns a static renderer for {@link Range} elements in which values have no units.
+	 * 
+	 * @param rangeResourceKeyPrefix
+	 *         the prefix of the translation key pointing to the range-based display text patterns
+	 * @param renderer
+	 *         a renderer used to display the plain value of a given {@link Range} bound
+	 * @see {@link #range(String, Renderer, Renderer)}
+	 */
+	public static <C extends Comparable<?>> Renderer<Range<C>> range(String rangeResourceKeyPrefix, Renderer<C> renderer) {
+		return range(rangeResourceKeyPrefix, renderer, renderer).nullsAsNull();
```

Comment:
I thing the call to `.nullsAsNull()` is useless here as it's already called in the `range` method you've called.